dcos_launch/platforms/aws.py

```diff
@@ -211,6 +211,7 @@ class CfStack:
     def __init__(self, stack_name, boto_wrapper):
         self.boto_wrapper = boto_wrapper
         self.stack = self.boto_wrapper.resource('cloudformation').Stack(stack_name)
+        self.name = stack_name
```

Comment:
there is already self.stack which has self.stack.stack_id, making this unnecessary